android/src/main/java/com/wix/reactnativenotifications/RNNotificationsPackage.java

```diff
@@ -26,7 +26,6 @@ public RNNotificationsPackage(Application application) {
         return Arrays.<NativeModule>asList(new RNNotificationsModule(mApplication, reactContext));
     }
 
-    @Override
     public List<Class<? extends JavaScriptModule>> createJSModules() {
```

Comment:
Why can't we just remove the whole function, if it is never used? Does it create any backward compatibility issue?

Reply:
This fix is needed urgently to upgrade to latest react-native version, maintainers kindly merge it and release soon.Thanks ...